sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -1012,6 +1179,13 @@ proces %2, a połączył się proces %3.%n%n
 
 Jeśli ta usługa jest skonfigurowana do uruchomienia z debugerem, takie zachowanie jest oczekiwane.
 .
+Language=Romanian
+Procesul serviciului, altul decât ceea ce este pornit în Administratorul de control al serviciilor
+e conectat odată cu pornirea serviciului %1. Administratorul de control al serviciilor a lansat, în schimb
+procesul %2 și procesul %3 conectat.%n%n
```

Comment:
This is still confusing and needs rework. Some hints:
- on first sentence, English tells that "a service process" "connected" (itself), whereas Romanian says "the service process" "is connected"
- on second sentence, English states clearly that "(SCM) launched process %2" (to connect) but "process %3 connected (itself) instead", whereas the Romanian sentence does not convey this (as clearly)
- it also may help to use reflexive when it's about that service/process that is connecting itself instead of the expected one